sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubClient/EventHubClientLiveTests.cs

```diff
@@ -215,5 +216,188 @@ await using (var client = new EventHubClient(connectionString))
                 }
             }
         }
+
+        /// <summary>
+        ///   Verifies that the <see cref="EventHubClient" /> is able to
+        ///   connect to the Event Hubs service.
+        /// </summary>
+        ///
+        [Test]
+        public async Task ClientCanRetrievePropertiesWithAmqpWebSocketsConnection()
```

Comment:
I'd recommend doing these as one test and using a `[TestCase]` with the `TransportType` since its the same test with just a varied bit of data fed in.

Reply:
Done!